weblate/trans/vcs.py

```diff
@@ -991,6 +991,25 @@ def check_config(self):
         """
         # We directly set config as it takes same time as reading it
         self.set_config('ui.ssh', ssh_file(SSH_WRAPPER))
+        self.set_config('ui.merge', 'internal:merge')
+
+        merge_driver = os.path.abspath(
+            os.path.join(
+                os.path.dirname(os.path.dirname(os.path.dirname(__file__))),
+                'examples',
+                'git-merge-gettext-po'
+            )
+        )
+
+        if os.path.exists(merge_driver):
```

Comment:
I've factored out this part from Git in 816e3f270, can you please update your code to use new method as well?